enterprise/server/cmd/ci_runner/main.go

```diff
@@ -149,7 +159,15 @@ func RunAllActions(ctx context.Context, cfg *workflowconf.BuildBuddyConfig) {
 		}
 		exitCode := 0
 		exitCodeName := "OK"
-		if err := ar.Run(ctx); err != nil {
+
+		// Include the repo's download time as part of the first invocation.
+		if !im.reported {
+			im.reported = true
+			ar.log.Printf("Fetched Git repository in %s\n", startTime.Sub(im.start))
+			startTime = im.start
+		}
+
+		if err := ar.Run(ctx, startTime); err != nil {
```

Comment:
Another way you could write this would be:

Inside of RunAllActions loop:
 if repo doesn't exist, clone it
 log the repo clone time

Then in your invocations you'll see a non-zero clone time for the first one and zero clone time for others.

Reply:
This was the first approach I wanted to go with, but it doesn't work because the RunAllActions loop is looping over an object that is sourced from the repo (the action config).

Reply:
Got it. OK.